test/column/editor.test.tsx

```diff
@@ -188,15 +220,34 @@ function EditorTest({ editable, editorOptions }: Pick<Column<Row>, 'editorOption
         key: 'col2',
         name: 'Col2',
         editable,
-        editor: TextEditor,
+        editor(p) {
```

Comment:
I've replaced `TextEditor`, as it closes on `blur` events, which is too good for our outside click tests.

Reply:
May be we can also remove `document.querySelector('input')` with `screen.get/qyeryByLabelText('col2-editor')`